### PR TITLE
Add metadata for Spectre Chain (id:39)

### PR DESCRIPTION
**Pull request by quan32**
Spectre Chain (SPECTRE) is a decentralized, high-efficiency and energy-saving public chain. It is compatible with smart contracts and supports high-performance transactions. The native token of SPECTRE is Spector Coin (SPC) and it adopts the PoS consensus mechanism.

More detail: https://head2head.gitbook.io/spectre-chain

Our homepage: https://spectre-bc.io/

Contact of development team
・Name: Quan Nguyen
・Email: quannt@bap.jp

### CODE REVIEW

File: _data/chains/eip155-39.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://spectre-bc.io/",
+  "shortName": "spc",
+  "chainId": 39,
+  "networkId": 39,
+  "explorers": [
+    {
+      "name": "Spectre Scan",
+      "url": "https://spectre-scan.io/",
```
Comment (ligi) [inline _data/chains/eip155-39.json:22]:
```suggestion
      "url": "https://spectre-scan.io",
```

**Comment by quan32** [comment]
@ligi I want to check status of this one.
Do I need to do anything else or just wait for your Team?

**Comment by quan32** [comment]
@ligi sorry but do you have any update?

**Comment by quan32** [comment]
@ligi sorry but can you help us update information on this one? What should we do next?

**Review by Uzocoiny — APPROVED** [review]

**Review by nutty323 — COMMENTED** [review]
Ok

**Review by pranver — APPROVED** [review]
Hi

**Pull request closed without merging**